src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './App.css';
+import './styles/output.css';
+import NavBar from './components/NavBar';
+import Manager from './components/Manager';
+import Footer from './components/Footer';
+function App() {
+  const [errors, setErrors] = useState({});
```

Comment:
I think this `errors` state can be removed. As fas as I understand, the errors are handled inside `Manager` component using `formErrors` state. Therefore, this might be not needed.

Can you check this?

Reply:
Yes, I will remove it since the errors are handled using the formErrors state within the Manager component.